app/controllers/api/functions.php

```diff
@@ -474,6 +475,38 @@
         ])));
 
         if ($next) {
+            $schedule = Authorization::skip(function () use ($dbForConsole, $project, $function) {
+                return $dbForConsole->findOne('schedules', [
```

Comment:
Yeh, I think it will be better to store the schedule ID in the function document and fetch it directly.